package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "./node_modules/.bin/eslint src/**/*.js --fix",
     "task1-1": "nodemon src/task1/task1-1",
     "task1-2": "nodemon src/task1/task1-2",
-    "task1-3": "nodemon --exec babel-node src/task1/task1-3"
+    "task1-3": "nodemon --exec babel-node src/task1/task1-3",
+    "task2": "npm run lint && nodemon --exec babel-node src/task2/server.js"
```

Comment:
I'm not sure that it's good idea to include lint check in execution command, usually it's necessary to place it for pre-commit or pre-push hooks

Reply:
Removed